opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -225,6 +225,12 @@
          */
         Scope startActive(boolean finishSpanOnClose);
 
+        /**
+         * @deprecated use {@link #startManual} or {@link #startActive} instead.
```

Comment:
Should be "@link #start" - and don't think this should reference startActive as is referring to the non-active behaviour.

Reply:
oh yes, fixed